components/credential-task/credential-task-directive.js

```diff
@@ -157,6 +173,26 @@ function brCredentialTaskDirective() {
       }
       return _.isEqual(query, CRYPTO_KEY_REQUEST);
     }
+
+    function _makePublic(credentials) {
+      var updatePromises = [];
+      for(var index in credentials) {
+        var credential = credentials[index];
+        var update = {
+          '@context': credential['@context'],
+          id: credential.id,
+          sysPublic: ['*']
+        };
+        var options = {};
+        if(!_compareHost(credential.id)) {
```

Comment:
Put a FIXME above this conditional that indicates that comparing the host and then modifying the URL on that basis is a hack that requires special knowledge and it needs to be fixed in the future.